lib/stripe/stripe_client.rb

```diff
@@ -513,7 +513,6 @@ def log_request(context, num_retries)
                      idempotency_key: context.idempotency_key,
                      query_params: context.query_params)
     end
-    private :log_request
```

Comment:
The `private` inline declarations were useless, there's already a `private` group keyword above: https://github.com/stripe/stripe-ruby/pull/786/commits/c89e19e082feb4799cfafd2b9305a81c98c739de#diff-48a74a0d322ee0e87e2be2e136f05b4bR197

Reply:
Change looks good, but we should consider just tagging everything with `private def method ...` in the same way we do internally. Way too easy to miss where the `private` block beings and ends.

Reply:
There's a linter for that! :) We can change the `EnforcedStyle` of linter [`Style/AccessModifierDeclarations`](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/AccessModifierDeclarations) from `group` (the default) to `inline`. It's also my personal preference, so I'll just add a commit to the PR.

Reply:
AWESOME. Haha.